app/Services/Adselect.php

```diff
@@ -67,7 +65,7 @@ public function addImpressions(array $events)
 
     public function getBanners(array $requests)
     {
-        return $this->rpcClient->execute('banner_select', $requests);
+//        return $this->rpcClient->execute('banner_select', $requests);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)